### ISSUE TITLE
WCAG 2.4.1: Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content (html)

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#3]
#### Issue
`Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content` ([bypass](https://dequeuniversity.com/rules/axe/3.5/bypass?application=msftAI))
#### Target application
[DASHBOARD - REGINFO.GOV](http://localhost:8080/public/jsp/Utilities/index.myjsp)
#### Element path
html
#### Snippet
`<html lang="en" class="">`
#### How to fix
    Fix any of the following:
      No valid skip link found
      Page does not have a header
      Page does not have a landmark region
#### Environment
Chrome version 81.0.4044.43
This accessibility issue was found using Accessibility Insights for Web 2.15.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
